main.js

```diff
@@ -18,8 +18,11 @@
  *        MIT License. All code unless otherwise specified is
  *        Copyright (c) Matthew Knox and Contributors 2015.
  */
-
+'use strict';
 // Load NodeJS Modifications/Variables
+require('babel-core');
+require('babel-register');
+require('babel-polyfill');
```

Comment:
Given these are modifying prototypes they should be in `prototypes.js` and so that /update works, require.once